app/javascript/components/search/results/ResultsPanel.js

```diff
@@ -15,38 +14,50 @@ const ResultsPanel = props => {
   const results = searchContext.results
   let panelContent
   if (searchContext.isError) {
-    panelContent =
-      <div className="error-panel">
-        <FontAwesomeIcon icon={faExclamationCircle}/>
-        Sorry, an error has occurred.  Support has been notified.
-        Please try again.
+    panelContent = (
+      <div className="error-panel  col-md-6 col-md-offset-3">
+        <FontAwesomeIcon
+          icon={faExclamationCircle}
+          style={{ 'margin-right': '5px' }}
+        />
+        Sorry, an error has occurred. Support has been notified. Please try
+        again. If this error persists, or you require assistance, please contact
+        support at{' '}
+        <a href="mailto:scp-support@broadinstitute.zendesk.com">
+          scp-support@broadinstitute.zendesk.com
+        </a>
```

Comment:
nice, I think this is the first time we've included the support email in a React page.  The second time, we'll probably want to move this into a lib component